src/plugins/intel_cpu/tests/unit/streams_info_table_test.cpp

```diff
@@ -390,924 +393,1568 @@ class StreamsCalculationTests : public CommonTestUtils::TestsCommon,
 
         std::vector<std::vector<int>> test_stream_info_table =
             ov::intel_cpu::get_streams_info_table(test_data.input_streams,
+                                                  test_data.input_streams_chaged,
                                                   test_data.input_threads,
                                                   test_data.input_infer_requests,
                                                   test_data.model_prefer_threads,
+                                                  test_data.input_perf_hint,
+                                                  test_data.scopeOflatencyCandidate,
                                                   test_data.proc_type_table);
 
         ASSERT_EQ(test_data.stream_info_table, test_stream_info_table);
     }
 };
 
-StreamsCalculationTestCase _2sockets_104cores_latency_1 = {
+StreamsCalculationTestCase _2sockets_104cores_latency_platform_1 = {
```

Comment:
Same ask as before: can we add corresponding commets to the test case?

Reply:
updated